include/osquery/logger.h

```diff
@@ -13,6 +13,11 @@
 #include <string>
 #include <vector>
 
+#ifdef WIN32
+#define GLOG_NO_ABBREVIATED_SEVERITIES
```

Comment:
A glog define of `ERROR` conflicts with windows.h: https://google-glog.googlecode.com/svn/trunk/doc/glog.html